serverless/eventing/tuning/override-config.adoc

```diff
@@ -13,6 +13,10 @@ You can override the default configurations for some specific deployments by mod
 The `replicas` spec cannot override the number of replicas for deployments that use the Horizontal Pod Autoscaler (HPA), and does not work for the `eventing-webhook` deployment.
 ====
 
+[NOTE]
+====
+You can override probes only for deployments that have a handler. Do not change the handler.
```

Comment:
This is not correct as user cannot change the handler as it is not exposed at the CRD level. 
The correct to say is users can override probes that are specified by default in the related deployment.
All Serving deployments define a readiness and a livessness probe by default which can be changed with the exception of:


- net-kourier-controller, 3scale-kourier-gateway that only define a readiness probe. Thus you can override only that.
- net-istio-controller has none can't override anything at the moment.
These last deployments will be covered by https://issues.redhat.com/browse/SRVKS-1046